.github/workflows/main.yml

```diff
@@ -35,8 +35,8 @@ jobs:
           echo Add other actions to build,
           echo test, and deploy your project.
           
-          # run super-linter
-          - name: Super-Linter
+         
+          - name: Super-Linter,
            uses: github/super-linter@v4.9.2
           
           
```

Comment:
- [ ] Kindly check this link to learn how to set up HTML and  CSS linters as they are not well configured in your project [HTML&CSS linters](https://github.com/microverseinc/linters-config/tree/master/html-css).